### ISSUE TITLE
Username color not displaying

### ISSUE THREAD

**Issue by Aceqxd** [sgtcaze/NametagEdit#568]
# Bug Report

**Type /ver NametagEdit and post the output.**
https://gyazo.com/7ad8f3af91f2fec591daf95b1cd11cd6

**What version of Bukkit/Spigot/PaperSpigot are you using? Type /ver**
https://gyazo.com/eace756027dcc56193cf1636cb9b9a7e

It is a fork of paper
**What plugins are you using? Type /plugins**
https://gyazo.com/3d8668e7f8888f8fd74eb5d47102df95

**Please explain your issue. How do you trigger it?**
Basically if I have more than 16 digits it will not show the username color in nametag but it will in tab since I have `LongTags: true`

https://gyazo.com/9f428441c09e47d68abc7941b1dd0d8c
What I set the prefix to `'&8[&9Sr.Mod&8] &9'`

**Are there any errors in the console? Please use: https://pastebin.com**
There are non

**Comment by Aceqxd** [comment]
I managed to find the issue, I was using nametag version 4.4.10 which was the newest version on spigot but apparently there is a 4.4.16 version which fixed the issue. 

Where I found the solution: https://github.com/sgtcaze/NametagEdit/issues/566
